code/datums/repositories/crew.dm

```diff
@@ -23,76 +23,56 @@ var/global/datum/repository/crew/crew_repository = new()
 		return cache_entry.data
 
 	var/tracked = scan()
-	for(var/obj/item/clothing/under/C in tracked)
+	for(var/t in tracked)
+		var/obj/item/clothing/under/C = t
 		var/turf/pos = get_turf(C)
 		if((C) && (C.has_sensor) && (pos) && (pos.z == z_level) && (C.sensor_mode != SUIT_SENSOR_OFF) && !within_jamming_range(C))
 			if(istype(C.loc, /mob/living/carbon/human))
 				var/mob/living/carbon/human/H = C.loc
 				if(H.w_uniform != C)
 					continue
 
-				var/list/crewmemberData = list("dead"=0, "area"="", "x"=-1, "y"=-1, "z"=-1, "level"="", "ref" = "\ref[H]")
+				var/list/crewmemberData = list("dead"=null, "area"=null, "x"=null, "y"=null, "z"=null, "level"=null, "ref" = "\ref[H]")
 
-				crewmemberData["sensor_type"] = C.sensor_mode
+				crewmemberData["stype"] = C.sensor_mode
 				crewmemberData["name"] = H.get_authentification_name(if_no_id="Unknown")
-				crewmemberData["rank"] = H.get_authentification_rank(if_no_id="Unknown", if_no_job="No Job")
-				crewmemberData["assignment"] = H.get_assignment(if_no_id="Unknown", if_no_job="No Job")
+				crewmemberData["ass"] = H.get_assignment(if_no_id="Unknown", if_no_job="No Job")
 
 				if(C.sensor_mode >= SUIT_SENSOR_BINARY)
 					crewmemberData["dead"] = H.stat > UNCONSCIOUS
 
 				if(C.sensor_mode >= SUIT_SENSOR_VITAL)
 					crewmemberData["pulse"] = "N/A"
-					crewmemberData["pulse_span"] = "neutral"
```

Comment:
Why's this gone? It helps with determining how fucked someone is at a glance.

Reply:
Because sending classes to UI many times is expensive and could slow down UI a lot. So server only needs to send data to carry just enough information to UI appropriately display it. So far span is just derived value from true pulse.